axelar-chains-config/info/testnet.json

```diff
@@ -1872,6 +1893,11 @@
       "gasOptions": {
         "gasLimit": 8000000
       },
+      "onchainGasEstimate": {
+        "chainName": "blast",
+        "l1ChainName": "ethereum",
+        "gasEstimationType": 1
```

Comment:
which params are appropriate for blast?

Reply:
We don't have the bedrock implemented in there, so just using OP as an upper bound.